### PR TITLE
ncat: add page

### PR DESCRIPTION
**Pull request by pollev**
ncat: create page

----

- [X] The page (if new), does not already exist in the repo.

- [X] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [X] The page has 8 or fewer examples.

- [X] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [X] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

**Comment by pollev** [comment]
My apologies for the incorrect title

**Comment by sbrl** [comment]
Thanks for the new page, @pollev! What's the difference between `ncat` and `nc`? I always get confused here :confused:

**Comment by pollev** [comment]
Oh shit. I don't think there is one. I did not know netcat had an abreviated version. My appologies. Feel free to remove the page

File: pages/linux/ncat.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# ncat
+
+> `cat` over networks.
+> Use the normal cat functionality over networks.
```
Comment (sbrl) [inline pages/linux/ncat.md]: Could these be combined into one line?

Comment (agnivade) [reply pages/linux/ncat.md]: Yes, I think the first line is redundant. And put cat within backticks.

File: pages/linux/ncat.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+- Listen for input on the specified port and write it to the specified file:
+
+`ncat -l {{port}} > /path/to/file`
```
Comment (agnivade) [inline pages/linux/ncat.md]: {{path/to/file}}

File: pages/linux/ncat.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+- Write output of specified file to the specified host on the specified port:
+
+`ncat {{address}} {{port}} < /path/to/file`
```
Comment (agnivade) [inline pages/linux/ncat.md]: ditto

**Comment by sbrl** [comment]
Thanks, @pollev! Sorry for the confusion :sweat_smile:

**Pull request merged**